README.md

```diff
@@ -1,710 +1,81 @@
-
-Data watchpoints may also be set, by clicking the + in the Watch section of the Debug side bar and typing the contract variable to watch in the format `<principal>.<contract>.<name>` or using the shortcut for a local contract, `.<contract>.<name>`. When a watchpoint is set on a contract variable, execution will pause when its value will change.
-
-![watchpoint](docs/images/watchpoint.png)
-
-During execution, the values of the current contract's variables, the current function's arguments, and any local variables (i.e. from a `let` expression) are shown in the side bar. The current watchpoints are also shown with their current values. In both cases, the contents of a map are not shown, but can be queried in the Debug Console. The call stack is also updated to show the call stack of the current execution.
-
-![view of side bar, showing variables, watchpoints, and call stack](docs/images/sidebar.png)
-
-At any point during execution, an expression can be evaluated in the current context via the Debug Console. Just type any valid Clarity expression and hit enter to evaluate it. Upon completion, the events emitted and the return value are printed to the debug console.
-
-![debug console](docs/images/debug-console.png)
-
-For more information on how you can use VS Code to debug smart contracts, please see the [How to Debug Smart Contracts in Clarinet’s VS Code Extension](https://www.youtube.com/watch?v=DsLCDQSijwk&list=PL5Ujm489LoJaAz9kUJm8lYUWdGJ2AnQTb&index=15) YouTube video.
-
-#### Command Line Debugger
-
-Inside of the console (`clarinet console`), there is a debugger for stepping through your contracts on the command line, including support for:
-
-- Breakpoints
-  - **Source**: Break at a specific line (and optional column) of a contract (`break` or `b` command)
-    ```
-    b SP466FNC0P7JWTNM2R9T199QRZN1MYEDTAR0KP27.miamicoin-token:28:4
-    ```
-  - **Function**: Break at a specific function (`break` or `b` command)
-    ```
-    b .crashpunks-v2.transfer
-    ```
-  - **Data**: Break on read/write to a variable or map (`watch` or `w` to break on write, `rwatch` or `rw` to break on read, and `awatch` or `aw` to break on read or write)
-    ```
-    w contracts/SP2KAF9RF86PVX3NEE27DFV1CQX0T4WGR41X3S45C.bitcoin-whales.payout
-    ```
-- Step execution
-  - **Step-in**: Step into the sub-expressions (`step` or `s` command)
-  - **Step-out**: Complete execution of the current expression and return the result back to the parent (`finish` or `f` command)
-  - **Step-over**: Continue to completion of the current expression, stepping over sub-expressions (`next` or `n` command)
-  - **Continue**: Continue execution until hitting a breakpoint or completing execution (`continue` or `c` command)
-- Evaluate and print the result of any expression (`print` or `p` command)
-
-To initiate a debug session, first enter the REPL console using the following command:
+# Check the contract
+clarinet check
 
-```
+# Launch the console
```

Comment:
i don't know if we should say "console" or "REPL"  here (lile, what will make the most sense for readers)

Reply:
since you added REPL above, i switched the language to match